resources/pod.py

```diff
@@ -275,3 +275,19 @@ def run_io_and_verify_mount_point(pod_obj, bs='10M', count='950'):
     mount_point = mount_point.split()
     used_percentage = mount_point[mount_point.index('/var/lib/www/html') - 1]
     return used_percentage
+
+
+def run_io_on_pool():
+    """
+    Run io on the pool
+
+    Returns:
+        Thread: A thread of the I/O execution
+    """
+
+    cmd = 'ceph osd lspools'
+    tools_pod = get_ceph_tools_pod()
+    pool = tools_pod.exec_ceph_cmd(ceph_cmd=cmd)
+    cmd1 = ('rados bench -p %s 60 write -t 32 -b 409600 --no-cleanup' % pool[0]['poolname'])
```

Comment:
```suggestion
    cmd1 = f"rados bench -p {pool[0]['poolname']} 60 write -t 32 -b 409600 --no-cleanup"
```

Reply:
You can check this https://github.com/red-hat-storage/ocs-ci/blob/master/tests/test_radosbench.py and move it as utility function so that others can reuse. Example usage is here https://github.com/red-hat-storage/ocs-ci/blob/master/tests/test_bench.py

Reply:
Done

Reply:
Noted.

Reply:
Done. Moved the function to tests/helpers.py